Signal/src/AppDelegate.m

```diff
@@ -719,6 +718,9 @@ - (void)storageIsReady
 
     [OWSPreferences setIsRegistered:[TSAccountManager isRegistered]];
 
+    // We need to do this _before_ anyone access the session or identity store state.
+    [OWSSessionStorage.sharedManager migrateFromStorageIfNecessary:TSStorageManager.sharedManager];
```

Comment:
This method is called `storageIsReady`, but how can storage be "ready" if we haven't migrated storage yet?

Is there a more intuitive name for this method?

Reply:
I've refactored the storage migration in a subsequent PR - it addresses this concern.